### ISSUE TITLE
'scrollingSpeed' property not working

### ISSUE THREAD

**Issue by jaimevent** [alvarotrigo/multiscroll.js#53]
I'm trying to set a scrolling speed, but even if I put a high value it doesn't change anything.
This is how I initiate the plugin:

``` javascript
$('#multiscroll').multiscroll({
    paddingTop: '15px',
    paddingBottom: '15px',
    loopBottom: true,
    loopTop: true,
    css3: true,
    scrollingSpeed: 2000,
    afterLoad: function(anchorLink, index) {
        if (timeout) {
            clearTimeout(timeout);
        }
        timeout = setTimeout(function () {
            $.fn.multiscroll.moveSectionDown();
        }, 3000);
    },
    afterRender: function() {
        if (timeout) {
            clearTimeout(timeout);
        }
        timeout = setTimeout(function () {
            $.fn.multiscroll.moveSectionDown();
        }, 3000);
    }
});
```

**Comment by alvarotrigo** [comment]
Scrolling speed only works when using `css3:false`. 

Otherwise you'll have to modify the `.ms-easing` class [in line 30](https://github.com/alvarotrigo/multiscroll.js/blob/master/jquery.multiscroll.css#L30) of the CSS file.

```csss
.ms-easing {
    -webkit-transition: all 0.7s ease-out;
    -moz-transition: all 0.7s ease-out;
    -o-transition: all 0.7s ease-out;
    transition: all 0.7s ease-out;
}
```

You can change the `0.7` seconds there for any other value you want.

**Comment by jaimevent** [comment]
Thanks, now it works!

**Comment by alvarotrigo** [comment]
I'll keep it open as an enhancement.

**Comment by alvarotrigo** [comment]
Fixed on the latest version multiScroll 0.2.3
